include/z64actor.h

```diff
@@ -283,7 +283,7 @@ typedef struct Player {
     /* 0x0678 */ u32        ageProperties;
     /* 0x067C */ u32        stateFlags1;
     /* 0x0680 */ u32        stateFlags2;
-    /* 0x0684 */ s32        unk_684;
+    /* 0x0684 */ Actor*	    unk_684;
```

Comment:
This is very minor but we use spaces over tabs for indentation
```suggestion
    /* 0x0684 */ Actor*     unk_684;
```